frontend/src/app/spot/components/text-field/stories/TextField.stories.mdx

```diff
@@ -119,7 +119,7 @@ The **Disabled** state shows all elements (including icons and any placeholder o
   </Story>
 </Canvas>
 
-The **Error** state puts a red outline around the text input. (_Note_: This state is only used when the text input is part of a _Form element_, which supplements this red outline with an message error message). The error state is usually temporary; fixing the cause of the error (via asyncronous form validation) usually allows it to revert back to the focus 
+The **Error** state puts a red outline around the text input. (_Note_: This state is only used when the text input is part of a _Form element_, which supplements this red outline with an message error message). The error state is usually temporary; fixing the cause of the error (via asyncronous form validation) usually allows it to revert back to the focus state.
```

Comment:
can also be synchronous form validation btw, so I'd just call it "via form validation"